### PR TITLE
EDNS(0): Trivial fixes

### PR DESCRIPTION
**Pull request by mkykadir**
I've missed some reviews in #2976 due to early merge :disappointed: 

See each commit.

### CODE REVIEW

File: source/agora/node/Registry.d [hunk 1]
```diff
@@ -413,7 +413,9 @@ public class NameRegistry: NameRegistryAPI
                         goto BAILOUT;
                     }
                     // Ignore the DO bit for now
-                    payloadSize = max(opt.payloadSize(), ushort(512));
+                    // `min` is to prevent DOS attack (request with huge payload size)
+                    payloadSize = min(max(opt.payloadSize(), ushort(512)),
+                        OPTRR.init.payloadSize());
```
Comment (Geod24) [inline source/agora/node/Registry.d]: Actually, looking at the code again, `responseOPT.payloadSize()` is more correct (same value in the end).
`OPT` is a way to advertise the max payload size for both clients and server. A server that would send a record with a payload higher than its own OPT `payloadSize()` would be buggy.

**Review by Geod24 — APPROVED** [review]
Just one nit.

File: source/agora/node/Registry.d (hunk 1 above)
Comment (mkykadir) [reply source/agora/node/Registry.d]: `payloadSize` is not responded back, always `responseOPT` is used and its `OPTRR.init`

Comment (Geod24) [reply source/agora/node/Registry.d]: Yes but if we ever change the OPT we respond, using `responseOPT.payloadSize()` will still be valid, while `OPTRR.init` will not.

Comment (mkykadir) [reply source/agora/node/Registry.d]: Updated

**Review by Geod24 — APPROVED** [review]
Merge at will

**Pull request merged**